README.md

```diff
@@ -5,6 +5,51 @@
 ![GitHub](https://img.shields.io/github/license/sultanshaikh-devops/DevOps-Course-Starter)
 ![GitHub last commit](https://img.shields.io/github/last-commit/sultanshaikh-devops/DevOps-Course-Starter)
 
+## Module 14
+----
+Application URL: http://127.0.0.1:5000/
+
+In this module we are using Minikube (is a version of Kubernetes).
+Following are required softwares.
+* Docker Desktop or Docker Engine
+* Kubectl
+* Minikube
+* Locally installed mongodb community edition
+
+Getting started:
++ Setting up oAuth on GitHub:
+  + Home page URL: http://127.0.0.1:5000
+  + Authorization callback URL: http://127.0.0.1:5000/login/callback
++ After mongodb installed create a database called todo_app and collection name lists.
++ Create a new docker image
+  ```bash
+  $ docker build --target production --tag todo-app:prod .
+  ```
++ start minikube
+  ```bash
+  $ minikube start
+  ```
++ Upload docker image
+  ```bash
+  $ minikube image load todo-app:prod
+  ```
+
++ Update the values before running, after the = sign. MONGO_CONNECTION_STRING value must inside " " as example below. This will load all required variables.  
```

Comment:
Thanks for spelling this out nice and clearly!
Worth noting that if you have a value that's predefined here though (like the collection name) then we might as well set it in the code, since it's not secret and doesn't need to be filled in by the user
Also remember that the local database depended on your host IP, so we would ideally provide some guidance for users to update that